src/main/java/net/openhft/chronicle/core/io/AbstractCloseable.java

```diff
@@ -242,11 +242,9 @@ public StackTrace createdHere() {
         return createdHere;
     }
 
-    /**
```

Comment:
why remove this javadoc ?

Reply:
Because it was moved to the interface. This way, all implementations will be documented.